spec/models/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  provider               :string           default("email"), not null
+#  uid                    :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  allow_password_change  :boolean          default(FALSE)
+#  remember_created_at    :datetime
+#  confirmation_token     :string
+#  confirmed_at           :datetime
+#  confirmation_sent_at   :datetime
+#  unconfirmed_email      :string
+#  name                   :string           not null
+#  nickname               :string
+#  image                  :string
+#  email                  :string           not null
+#  tokens                 :json
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+require 'rails_helper'
+
+describe User, type: :model do
+  subject { build(:user) }
+
+  it { should validate_presence_of(:name); }
```

Comment:
Using `should` is deprecated, instead use `is_expected` 
Check here: http://www.betterspecs.org/#expect